src/Aspire.Dashboard/Components/Controls/GridValue.razor.cs

```diff
@@ -82,4 +84,9 @@ private string TrimLength(string? text)
 
         return text ?? "";
     }
+
+    public void Dispose()
+    {
+        _tooltipComponent?.Dispose();
```

Comment:
Why is this necessary? There are other places we're using tooltip and not disposing.

Reply:
This will close the tooltip. I could only reproduce the issue you filed about the tooltips not closing on data grid data update on tooltips located in the data grid itself. So I don’t think it’s necessary elsewhere

Reply:
Which issue is that? I'd like to test it works.

There should be a comment saying why it's being disposed.

Reply:
This one: https://github.com/dotnet/aspire/issues/2704

Test on the stress project. The resources that update have this behavior.

Reply:
I am unable to run the stress project. Starting some of the projects fails